README.md

```diff
@@ -41,17 +26,17 @@ Because a user's folder only holds items that they own, it effectively checks al
 ## Environment
 
 1. Clone the ArcGIS Pro conda environment:
-   - `conda create --clone arcgispro-py3 --name validator`
+   * `conda create --clone arcgispro-py3 --name validator`
```

Comment:
aww i like the `-`'s

Reply:
My markdown linter was yelling at me :)

Reply:
Burn that linter! I use markdownlint and like it.

Reply:
So am I, but I figured out why it wasn't happy: I created a list above with *, so it wanted every other list to be * as well. Next project will be -'s.